src/Data/Fin/Base.agda

```diff
@@ -143,6 +143,11 @@ quotRem {suc n} k i with splitAt k i
 ... | inj₁ j = j , zero
 ... | inj₂ j = Product.map₂ suc (quotRem {n} k j)
 
+-- inverse of above function
+combine : ∀ {n k} → Fin k → Fin n → Fin (n ℕ.* k)
+combine x zero = inject+ _ x
+combine x (suc y) = raise _ (combine x y)
```

Comment:
Can we fill in the `_` here? Even if it means we have to add the implicit arguments, I think it'll make it more readable?